modelmapper/cleaner.py

```diff
@@ -60,16 +72,35 @@ def get_csv_data_cleaned(self, path_or_content, original_content_type=None):
             try:
                 field_info = model_info[field_name]
             except KeyError:
-                raise KeyError(FIELD_NAME_NOT_FOUND_MSG.format(field_name)) from None
+                if ignore_new_fields:
+                    self._missing_fields.add(field_name)
+                    continue
+                else:
+                    raise KeyError(FIELD_NAME_NOT_FOUND_MSG.format(field_name)) from None
```

Comment:
I know you just moved the existing line, but you can take out that `from None` part. It's unnecessary.